utils/principal_curvature.py

```diff
@@ -198,7 +199,7 @@ def get_shape_operator_isoresponse_surface(pt_grad, pt_hess):
     
     # make sure that our normal points in the direction of the function gradient
     pt_grad = pt_grad.clone()
-    pt_grad[-1] = -torch.abs(pt_grad[-1])
+    #pt_grad[-1] = -torch.abs(pt_grad[-1])
```

Comment:
@matthias-k is this going to change the curvature calculation for the non-projected case?

Reply:
@dpaiton no, it's now taking care of and the end of the function by inverting grad_g and hess_g if needed.